nislmigrate/migrators/file_migrator.py

```diff
@@ -118,19 +118,25 @@ def restore(self, migration_directory: str, facade_factory: FacadeFactory, argum
             configuration.mongo_configuration,
             migration_directory,
             self.name)
-
-        if configuration.should_update_store:
-            self.update_root_file_path_in_metadata(configuration)
-
-        if configuration.use_forward_slashes:
-            self.update_file_path_slashes_in_metadata(configuration)
-
+        self.modify(self, migration_directory, facade_factory, arguments)
```

Comment:
I think it would be organizationally better to call a shared private method rather than directly calling the public modify here.

Reply:
These is now a `update_database` helper function that is called by both restore and modify.